### ISSUE TITLE
problem with qt 6.6.2

### ISSUE THREAD

**Issue by iltommi** [MeVisLab/pythonqt#194]
I get these 3 errors when compiling pythonqt since I upgraded to Qt 6.6.2 (on macos with qt installed with homebrew) 
```
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -c -pipe -stdlib=libc++ -Wno-deprecated-declarations -Wuninitialized -Winit-self -pedantic -I/usr/local/opt/python@3.12/Frameworks/Python.framework/Versions/3.12/include/python3.12 -I/usr/local/opt/python@3.12/Frameworks/Python.framework/Versions/3.12/include/python3.12 -O2 -std=c++1z  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -mmacosx-version-min=12.0 -Wall -Wextra -fPIC -DPYTHONQT_QTALL_EXPORTS -DPYTHONQT_WITH_CORE -DPYTHONQT_WITH_GUI -DPYTHONQT_WITH_SVG -DPYTHONQT_WITH_SQL -DPYTHONQT_WITH_NETWORK -DPYTHONQT_WITH_XML -DPYTHONQT_WITH_MULTIMEDIA -DPYTHONQT_WITH_QML -DPYTHONQT_WITH_QUICK -DPYTHONQT_WITH_UITOOLS -DPYTHONQT_WITH_WEBENGINEWIDGETS -DQT_NO_DEBUG -DQT_WEBENGINEWIDGETS_LIB -DQT_PRINTSUPPORT_LIB -DQT_SVGWIDGETS_LIB -DQT_MULTIMEDIAWIDGETS_LIB -DQT_QUICKWIDGETS_LIB -DQT_UITOOLS_LIB -DQT_OPENGLWIDGETS_LIB -DQT_WIDGETS_LIB -DQT_SVG_LIB -DQT_MULTIMEDIA_LIB -DQT_WEBENGINECORE_LIB -DQT_QUICK_LIB -DQT_OPENGL_LIB -DQT_GUI_LIB -DQT_SQL_LIB -DQT_QMLMODELS_LIB -DQT_WEBCHANNEL_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_XML_LIB -DQT_QMLINTEGRATION_LIB -DQT_POSITIONING_LIB -DQT_CORE_LIB -I. -I../../src -I/usr/local/lib/QtWebEngineWidgets.framework/Headers -I/usr/local/lib/QtPrintSupport.framework/Headers -I/usr/local/lib/QtSvgWidgets.framework/Headers -I/usr/local/lib/QtMultimediaWidgets.framework/Headers -I/usr/local/lib/QtQuickWidgets.framework/Headers -I/usr/local/lib/QtUiTools.framework/Headers -I/usr/local/lib/QtOpenGLWidgets.framework/Headers -I/usr/local/lib/QtWidgets.framework/Headers -I/usr/local/lib/QtSvg.framework/Headers -I/usr/local/lib/QtMultimedia.framework/Headers -I/usr/local/lib/QtWebEngineCore.framework/Headers -I/usr/local/lib/QtQuick.framework/Headers -I/usr/local/lib/QtOpenGL.framework/Headers -I/usr/local/lib/QtGui.framework/Headers -I/usr/local/lib/QtSql.framework/Headers -I/usr/local/lib/QtQmlModels.framework/Headers -I/usr/local/lib/QtWebChannel.framework/Headers -I/usr/local/lib/QtQml.framework/Headers -I/usr/local/lib/QtNetwork.framework/Headers -I/usr/local/lib/QtXml.framework/Headers -I/usr/local/include -I/usr/local/include/QtQmlIntegration -I/usr/local/lib/QtPositioning.framework/Headers -I/usr/local/lib/QtCore.framework/Headers -I. -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk/System/Library/Frameworks/OpenGL.framework/Headers -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk/System/Library/Frameworks/AGL.framework/Headers -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk/System/Library/Frameworks/OpenGL.framework/Headers -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk/System/Library/Frameworks/AGL.framework/Headers -I/usr/local/share/qt/mkspecs/macx-clang -F/usr/local/lib -o com_trolltech_qt_gui8.o ../../generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui8.cpp
In file included from ../../generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui5.cpp:1:
../../generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui5.h:1815:71: error: invalid parameter name: 'static_assert' is a keyword
void py_set_static_assert(QMessageBox* theWrappedObject, QMessageBox  static_assert){ theWrappedObject->static_assert = static_assert; }
                                                                      ^
../../generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui5.h:1815:105: error: expected unqualified-id
void py_set_static_assert(QMessageBox* theWrappedObject, QMessageBox  static_assert){ theWrappedObject->static_assert = static_assert; }
                                                                                                        ^
../../generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui5.h:1816:92: error: expected unqualified-id
QMessageBox  py_get_static_assert(QMessageBox* theWrappedObject){ return theWrappedObject->static_assert; }
```
Could someone help me?

**Comment by mrbean-bremen** [comment]
Should be fixed with the latest version (see #192).

**Comment by mrbean-bremen** [comment]
Closing as fixed, please reopen if it doesn't work for you with current master.
